src/index.ts

```diff
@@ -557,7 +557,13 @@ export function getConfigUrl(apiKey: string, baseUrl?: string): URL {
  * @public
  */
 export class EppoPrecomputedJSClient extends EppoPrecomputedClient {
-  public static instance: EppoPrecomputedJSClient;
+  public static instance = new EppoPrecomputedJSClient({
```

Comment:
just to confirm, this instantiation doesn't kick off any network requests or polling, right?

Reply:
Nope, you have to go through the `precomputedInit` method to start the poller